default/stringtables/en.txt

```diff
@@ -7935,57 +7943,116 @@ Launch Bay
-Each launch bay allows a number of fighters to launch each combat round.
-'''
+Launched fighters join combat the round after being launched, returning once combat ends for that turn.'''
 
 # Ship part classes
 PC_SHIELD
 Shields
 
+PC_SHIELD_DESC
+'''Shield parts reduce the [[encyclopedia DAMAGE_TITLE]] sustained from each weapon hit by the shield parts strength.
+
+Ship shields are not affected by weapon hits, and reduce all subsequent direct fire weapon hits' damage by the same amount. Ship shields are ineffective against [[encyclopedia FIGHTER_TECHS]].'''
+
 # Ship part classes
 PC_ARMOUR
 Armor
 
+PC_ARMOUR_DESC
+Armor parts increase the [[metertype METER_STRUCTURE]] of a ship, allowing a ship to sustain more damage before being destroyed.
+
 # Ship part classes
 PC_TROOPS
 Ground Troops
 
+PC_TROOPS_DESC
+'''Troop parts increase the [[metertype METER_TROOPS]] on-board, which can be used to invade enemy planets.
+
+To successfully invade a planet, the total number of [[metertype METER_TROOPS]] from all invading ships must exceed the total defending (ties are awarded to the defender).
+An invading ship is destroyed as part of the landing operation (any additional parts on the ship are lost as well).'''
+
 # Ship part classes
 PC_DETECTION
 Detection
 
+PC_DETECTION_DESC
+'''Detection parts increase the [[metertype METER_DETECTION]] for the ship they are equipped on.
+
+An empire's [[encyclopedia DETECTION_TITLE]] must equal or exceed an object's [[metertype METER_STEALTH]] to detect the object.'''
+
 # Ship part classes
 PC_STEALTH
 Stealth
 
+PC_STEALTH_DESC
+'''Stealth parts increase the [[metertype METER_STEALTH]] of a ship. The effects from these parts do not complement each other, only the highest rated part will affect the ship.
+
+If combat occurs in a system, ships that are invisible to an enemy are not targetted by the enemy's weapons. If the ship attacks, it may become visible for the following rounds.
+
+See also: [[metertype METER_DETECTION]], [[encyclopedia DETECTION_TITLE]]'''
+
 # Ship part classes
 PC_FUEL
 Fuel
 
+PC_FUEL_DESC
+'''Fuel parts affect a ship's [[metertype METER_FUEL]]. Such parts might increase the maximum fuel capacity or increase the amount of fuel the ship can scavenge or regenerate.
+
+A ship requires a fuel value of 1.0 or more to leave a system.'''
+
 # Ship part classes
 PC_COLONY
 Colonization
 
+PC_COLONY_DESC
+'''Colony parts integrate with a ship to provide the necessary equipment to establish ownership on a new planet.
+
+These parts may also increase the [[metertype METER_POPULATION]] on a ship.
+When colonizing a planet with a populated colony ship, a colony for the [[encyclopedia ENC_SPECIES]] is established, starting with the same population that was aboard the ship.
+
+A ship with no population will still have enough personnel to establish an [[encyclopedia OUTPOSTS_TITLE]].
+Outposts may construct a colony for any empire species within supply range, provided the source meets other criteria.
```

Comment:
Meant that I have changed to such.  This should also link to a `Colony Building` category/article in #1347 , supplementing or replacing `provided the source meets other criteria`.